docs/search_syntax.md

```diff
@@ -0,0 +1,228 @@
+| ```archived``` | True if run is archived, otherwise False | ```boolean```|
+| ```active``` | True if run is active(in progress), otherwise False | ```boolean``` |
+| ```duration``` | Run duration in seconds | ```numeric```|
+| ```created_at``` | Run creation datetime | ```numeric``` |
+| ```finalized_at``` | Run end datetime | ```numeric``` |
+| ```metrics``` | Set of run metrics | ```dictionary``` |
+
+## Search Metrics
+You can filter the metrics using the following metric attributes:
+| Property    | Type |
+| -------- | ------- |
+| ```name```  | ```string```|
+| ```last``` | ```numeric```|
+| ```last_step``` | ```numeric```|
+| ```first_step``` | ```numeric```|
+
+### String operations
+For the ```string``` attributes you can use the following comparing operator:
+- ``` == ```
+- ``` != ```
+- ``` in ```
+- ``` .startswith() ```
+- ``` .endswith() ```
+
+### Numeric operations
+For the ```numeric``` attributes you can use the following comparing operator:
+- ``` == ```
+- ``` != ```
+- ``` > ```
+- ``` >= ```
+- ``` < ```
+- ``` <= ```
+
+### Boolean operations
+For the ```boolean``` attributes you can use the following comparing operator:
+- ``` == ```
+- ``` != ```
+
+### Logical operations
+You can create complex search queries combining multiple conditions with logical operators.
+- ``` and ```
+- ``` or ```
+- ``` not ```
+
+
+#### Example with ```run.name``` (string)
+Select only the runs where the name exactly matches "TestRun1"
+
+```python
+run.name == "TestRun1"
+```
+
+Select only the runs where the name is different from "TestRun1"
+```python
+run.name != "TestRun1"
+```
+
+Select only the runs where "Run1" is contained in the run name
+```python
+"Run1" in run.name
+```
+
+Select only the runs where the name starts with "Test"
+```python
+run.name.startswith('Test')
+```
+
+Select only the runs where the name ends with "Run1"
+```python
+run.name.endswith('Run1')
+```
+
+#### Example with ```run.duration``` (numeric)
+
+Select only the runs where the duration is exactly 111111111
+
+```python
+run.duration == 111111111
+```
+
+
+Select only the runs where the duration is not 111111111
+```python
+run.duration != 111111111
+```
+
+Select only the runs where the duration is greater than 111111111
+```python
+run.duration > 111111111
+```
+
+Select only the runs where the duration is greater or equal to 111111111
+```python
+run.duration >= 111111111
+```
+
+Select only the runs where the duration is less than 111111111
+```python
+run.duration < 111111111
+```
+
+Select only the runs where the duration is less or equal to 111111111
+```python
+run.duration <= 111111111
+```
+
+#### Example with ```run.archived``` (boolean)
+Select only the runs where the archived attribute is true
+```python
+run.archived == True
+```
+
+Select only the runs where the archived attribute is not true
+```python
+run.archived != True
+```
+
+### Run parameters
+Run parameters could be accessed via attributes.
+```python
+run.hparams.param1 >= 64
```

Comment:
typo? `hparams`? should be just `params`.

Reply:
Actually should be just param1 I'll fix that